symengine/expression.h

```diff
@@ -32,16 +32,16 @@ class Expression
     //! Construct Expression from `int`
     Expression(int n) : m_basic(integer(n)) {}
     //! Construct Expression from Basic
-    template <typename T, typename = typename std::enable_if<std::is_constructible<RCP<const Basic>, T &&>::value>::type>
+    template <typename T>
```

Comment:
@bluescarni thanks for the feedback. I'll try again, maybe the error was caused by something else than `noexpect`.

It's always a compromise between the very latest features and being able to compile on all platforms with various compilers. I would like to compile with gcc, clang, Intel and MSVC. That will ensure that the code is truly platform independent, in practical sense.